README.md

```diff
@@ -1,6 +1,6 @@
 <picture>
   <source media="(prefers-color-scheme: dark)" srcset="./gui/public/StanPlay_Logo_RGB_White.png">
-  <img alt="The Stan Playground logo" src="./gui/public/StanPlay_Logo_RGB_Full.png" align="right" width=25%>
+  <img alt="The Stan Playground logo" src="./gui/public/StanPlay_Logo_RGB_Full.png" style="float: right;" width=25%>
```

Comment:
This didn't work: https://github.com/flatironinstitute/stan-playground/blob/readme-tweak-js/README.md

Reply:
Reverted. We are still getting the HR cutting through the logo (at least in my browser) but I'm just going to leave it for the time being.